Project.toml

```diff
@@ -5,6 +5,7 @@ version = "0.1.0"
 
 [deps]
 BlockDiagonals = "0a1fb500-61f7-11e9-3c65-f5ef3456f9f0"
+ChainRulesCore = "d360d2e6-b24c-11e9-a2a3-2a2ae2dbcce4"
```

Comment:
Why not just as a docs dependency?

Reply:
1. we already depend on it transitively through MathOptSetDistances
2. At some point we will use ChainRulesCore instead of querying the projection Jacobian from MathOptSetDistances